.rubocop.yml

```diff
@@ -26,6 +26,9 @@ Layout/EmptyLinesAroundClassBody:
 Layout/IndentHash:
   Enabled: false
 
+Layout/ExtraSpacing:
+  Enabled: true
```

Comment:
This is enabled by default, I believe https://rubocop.readthedocs.io/en/latest/cops_layout/#layoutextraspacing

Reply:
Ahh, you're right